uaclient/apt_news.py

```diff
@@ -263,11 +263,19 @@ def local_apt_news(cfg: UAConfig) -> Optional[str]:
 
 
 def format_news_for_apt_update(news: str) -> str:
-    result = "#\n"
-    for line in news.split("\n"):
-        result += "# {}\n".format(line)
-    result += "#\n"
-    return result
+    lines = news.split("\n")
+    formatted_lines = []
+
+    if system.get_release_info().series == "oracular":
+        prefix = "\nAPT news:"
+        for line in lines:
+            formatted_lines.append("  " + line)
+        return "{0}\n{1}\n\n".format(prefix, "\n".join(formatted_lines))
+    else:
+        prefix = "#"
+        for line in lines:
+            formatted_lines.append(prefix + " " + line)
+        return "{0}\n{1}\n{0}\n".format(prefix, "\n".join(formatted_lines))
```

Comment:
I think this is still less clear than something like the original. For example:

```suggestion
    if system.get_release_info().series == "oracular":
        result = "\nAPT news:"
        for line in lines:
            result += "  {}\n".format(line)
        result += "\n"
        return result
    else:
        result = "#\n"
        for line in lines:
            result += "# {}\n".format(line)
        result += "#\n"
        return result
```

Reply:
the string formatting was the issue, making it a hard read here, apologies. Fixed!